pkg/auth/common/token.go

```diff
@@ -20,6 +20,8 @@ const (
 	extraIDToken = "id_token"
 )
 
+var currentTime = time.Now
```

Comment:
Would you mind adding a comment to explain that `currentTime` is used to allow tests to override it?

Reply:
As will be removed and addressed as part of https://github.com/vmware-tanzu/tanzu-cli/pull/820#discussion_r1790816475 by using separate package.